R/oauth-token.r

```diff
@@ -141,8 +141,8 @@ Token <- R6::R6Class("Token", list(
 oauth1.0_token <- function(endpoint, app, permission = NULL,
                            as_header = TRUE,
                            private_key = NULL,
-                           cache = getOption("httr_oauth_cache")) {
-  params <- list(permission = permission, as_header = as_header)
+                           cache = getOption("httr_oauth_cache"), use_oob = getOption("httr_oob_default")) {
```

Comment:
Also need a newline, and should probably switch list to use tidy style: http://style.tidyverse.org/syntax.html#long-lines